### PR TITLE
Feat/eae 39 overview workflows

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: src/components/CreateWorkflow.vue [hunk 1]
```diff
@@ -54,6 +54,19 @@
         Create Workflow
       </v-btn>
     </v-row>
+
+    <v-container v-if="createForm === false">
+      <v-alert outlined color="primary">
+        Scehmatization created!
```
Comment (SiggyF) [inline src/components/CreateWorkflow.vue]:
```suggestion
        Schematization created!
```

File: src/components/ListWorkflows.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <v-list-item-title v-text="workflow.title"></v-list-item-title>
+      </v-list-item-content>
+      </v-list-item>
+    </v-list>
+  </div>
+</template>
+
+<script>
+const colors = {
+  running: '#001FFF',
```
Comment (SiggyF) [inline src/components/ListWorkflows.vue:18]: stylesheet anyone?

File: src/components/MapComponent.vue [hunk 3]
```diff
@@ -22,15 +22,16 @@ import { mapState, mapMutations } from 'vuex'
...
-  watch: {
-    $route (to, from) {
-      if (to.name === 'Editor') {
-        this.addDrawingTools()
-      } else if (from.name === 'Editor') {
-        this.removeDrawingTools()
-      }
-    }
-  },
+  // watch: {
```
Comment (SiggyF) [inline src/components/MapComponent.vue]: Remove

File: src/router/index.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ const routes = [
     name: 'Home'
   },
   {
-    path: '/Editor',
+    path: '/Editor/:jobId',
```
Comment (SiggyF) [inline src/router/index.js]:
```suggestion
    path: '/editor/:jobId',
```

File: src/views/Editor.vue [hunk 5]
```diff
@@ -1,43 +1,52 @@
...
-        name="input-7-4"
-        label="Solo textarea"
-        :value="createdMessage"
-      ></v-textarea>
+      <div class="flex-grow-1 editor-content">
+        <list-workflows v-if="$route.params.jobId === 'list'" />
+        <create-workflow v-if="$route.params.jobId === 'new'" />
+        <workflow-status v-if="$route.params.jobId !== 'new' && $route.params.jobId !== 'list' && $route.params.jobId" />
+      </div>
+      <v-btn to="/Editor/new" class="my-3 flex-grow-0"  outlined block>
```
Comment (SiggyF) [inline src/views/Editor.vue]:
```suggestion
      <v-btn to="/editor/new" class="my-3 flex-grow-0"  outlined block>
```

File: src/views/Editor.vue [hunk 6]
```diff
@@ -1,43 +1,52 @@
...
+        Create new workflow
+      </v-btn>
+      <v-btn to="/Editor/list" class="flex-grow-0"  outlined block>
```
Comment (SiggyF) [inline src/views/Editor.vue]:
```suggestion
      <v-btn to="/editor/list" class="flex-grow-0"  outlined block>
```

File: src/views/Editor.vue (hunk 5 above)
Comment (SiggyF) [inline src/views/Editor.vue]:
```suggestion
      <v-btn :to="{ name: 'editor', params: { jobId: 'new' }}" class="my-3 flex-grow-0"  outlined block>
```

**Review by SiggyF — APPROVED** [review]
Make sure you always use lowercase url's.

File: src/components/ListWorkflows.vue (hunk 2 above)
Comment (CindyvdVries) [reply src/components/ListWorkflows.vue:18]: Yes! am creating that for the entire project in new branch! will take these up in there as well

**Pull request merged**